composer.json

```
@@ -19,10 +19,9 @@
             "Afterpay\\SDK\\Test\\": [ "test" ]
         }
     },
-    "require": {
-    },
     "require-dev": {
-        "squizlabs/php_codesniffer": "^3.5"
+        "squizlabs/php_codesniffer": "^3.5",
+        "phpunit/phpunit": "^5.0||^6.0||^7.0||^8.0||^9.0"
```

Shouldn't these be in the opposite order? Ordered from latest/newest first to earliest/oldest last?

---

Apparently the order doesn't matter. Can we add spaces for readability though please?

```suggestion
        "phpunit/phpunit": "^5.0 || ^6.0 || ^7.0 || ^8.0 || ^9.0"
```